src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -377,7 +597,8 @@ private static bool IsSafeForReordering(BoundExpression expression, RefKind kind
             bool expanded,
             ImmutableArray<int> argsToParamsOpt,
             ref ImmutableArray<RefKind> argumentRefKindsOpt,
-            out ImmutableArray<LocalSymbol> temps,
+            [NotNull] ref ArrayBuilder<LocalSymbol>? temps,
+            BitVector positionsAssignedToTemp,
```

Comment:
>positionsAssignedToTemp [](http://example.com/codeflow?start=22&length=23)

Does this really make an observable difference?

Reply:
It should have been preventing duplicate stores, but in the next revision it will also be necessary to prevent stores from being optimized away when they shouldn't be.

Reply:
Actually, after thinking about this more, I think we can get rid of this and decouple `Visit` and `Make` a bit more.